### PR TITLE
Add distro logos on nodes page

### PR DESCRIPTION
**Pull request by aidanharris**
Some examples:

Debian and Ubuntu:

Arch and Gentoo:

Raspbian and Alpine:

### CODE REVIEW

**Comment by rusher81572** [comment]
@aidanharris Thanks. I will review the code changes today hopefully. Looking at the screenshots, it looks like there needs to be more spacing between nodes. The second node output looks too close to the first nodes.I love the Raspberry pi logo =)

File: web/nodes.html [hunk 1]
```diff
@@ -16,14 +17,11 @@
 				for (i = 0, dist = 0; i != test.length; i++) {
 					if (test[i].length > 2) {
 						if (test[i].indexOf('Node:') > -1) {
-							document.write('<br>' + distLogo(dists[dist++]) + '<br><p align=left><br><font size=+2>' + test[i] + '<br>'+ test[i+1] + '<br>'+ test[i+2] + '<br><br>'+ test[i+3] + '<br><br>' + test[i+4] + "</font>");
+							document.write('<br><br><p align=left>' + distLogo(dists[dist++]) + '<br><font size=+2>' + test[i] + '<br>'+ test[i+1] + '<br>'+ test[i+2] + '<br><br>'+ test[i+3] + '<br><br>' + test[i+4] + "</font>");
```
Comment (aidanharris) [inline web/nodes.html]: @rusher81572 I've fixed the spacing now, the logos should now take up the exact same space the server image did.

**Comment by rusher81572** [comment]
Thanks for the PR. Here are a few issues I found:

1. View Running

```
container-layout.html:20 Uncaught TypeError: Cannot read property 'split' of undefined
    at Object.success (container-layout.html:20)
    at fire (jquery-1.10.2.js:3048)
    at Object.fireWith [as resolveWith] (jquery-1.10.2.js:3160)
    at done (jquery-1.10.2.js:8235)
    at XMLHttpRequest.callback (jquery-1.10.2.js:8778)
﻿```

2. View Nodes 
```
Node: /bin/sh: nodejs: command not found
Node: /bin/sh: nodejs: command not found
```

**Comment by rusher81572** [comment]
Checked out the latest changes.

```
container-layout.html:20 Uncaught TypeError: Cannot read property 'split' of undefined
    at Object.success (container-layout.html:20)
    at fire (jquery-1.10.2.js:3048)
    at Object.fireWith [as resolveWith] (jquery-1.10.2.js:3160)
    at done (jquery-1.10.2.js:8235)
    at XMLHttpRequest.callback (jquery-1.10.2.js:8778)
```
```
image-layout.html:20 Uncaught TypeError: Cannot read property 'split' of undefined
    at Object.success (image-layout.html:20)
    at fire (jquery-1.10.2.js:3048)
    at Object.fireWith [as resolveWith] (jquery-1.10.2.js:3160)
    at done (jquery-1.10.2.js:8235)
    at XMLHttpRequest.callback (jquery-1.10.2.js:8778)
﻿```

Also, I think we should add logic to detect  if it should use node or nodejs to be more crossplatform

**Comment by rusher81572** [comment]
I did the following hoping to get around these errors:

```
cd /usr/local/bin
 ln -s node nodejs
```

And got weird errors when viewing nodes:
```
Node: module.js:487
throw err;
^
```

**Comment by aidanharris** [comment]
@rusher81572 I think you should stick to assuming "node" exists and here's my hopefully compelling argument why...

Everyone but Debian and Ubuntu (see upstreams tarball for example: https://nodejs.org/dist/v6.11.1/node-v6.11.1-linux-x64.tar.gz) uses node not nodejs. On Ubuntu and Debian systems there is the nodejs-legacy package that should correct this (https://packages.debian.org/sid/nodejs-legacy) (https://packages.ubuntu.com/search?keywords=nodejs-legacy). Effectively what Debian has done is the equivalent of renaming /bin/sh to /bin/bannana and expecting there to be no issues. If you add the need to install the nodejs-legacy package to the README we should be okay to assume that the node binary exists instead of having to come up with some elaborate (but totally possible) workaround.

**Comment by rusher81572** [comment]
@aidanharris, Ok, once all the bugs are fixed in this PR, I will update the README.

**Comment by aidanharris** [comment]
@rusher81572 When you had that npm install error were you running the install as root? I managed to reproduce it on my pi2 and it stopped as soon as I ran the install as a regular user. Other than that I have no idea what's causing the problem. It definitely is an npm bug / error and nothing to do with me. I'm still doing testing now to try to figure out what's going on. I wish npm wasn't so bad maybe we should try the installation with [yarn](https://yarnpkg.com) instead and see if it still errors.

**Comment by rusher81572** [comment]
On the Mac, I am not running it as root. I removed node_modules and now I see the getos.
I think it is ready to merge unless you have any objections.

**Comment by rusher81572** [comment]
@aidanharris oh, please change the version number to 1.9.6 in main.html.

**Comment by rusher81572** [comment]
Thank you very much for all the hard work. Appreciate it

**Comment by aidanharris** [comment]
@rusher81572 I've just completed a fresh installation on Raspbian on my Pi2 with no issues whatsoever so am happy for you to merge this now (never mind you've already done it 😄 . Would you mind enabling the Wiki so that I can write up install instructions as I did a couple of perhaps non-standard things such as updating docker to the latest version and updating nodejs to the latest lts as both versions are outdated in Raspbian.

**Comment by rusher81572** [comment]
Sure, enabled =)

**Pull request merged**